src/state/farms/index.ts

```diff
@@ -61,7 +61,7 @@ export const fetchFarmsPublicDataAsync = () => async (dispatch, getState) => {
   const fetchArchived = getState().farms.loadArchivedFarmsData
   const farmsToFetch = fetchArchived ? farmsConfig : nonArchivedFarms
   const farms = await fetchFarms(farmsToFetch)
-  const farmsWithPrices = await fetchFarmsPrices(farms)
+  const farmsWithPrices = await fetchFarmsPrices(farms, getState)
```

Comment:
Would it be better to get prices the line before and pass them as 2nd argument instead of `getState` here?